projects/packages/my-jetpack/_inc/components/welcome-flow/index.tsx

```diff
@@ -20,10 +20,19 @@ export type WelcomeFlowExperiment = {
 	variation: 'control' | 'treatment';
 };
 
-const WelcomeFlow: FC< PropsWithChildren > = ( { children } ) => {
+interface Props extends PropsWithChildren {
+	welcomeFlowExperiment: WelcomeFlowExperiment;
+	setWelcomeFlowExperiment: React.Dispatch< React.SetStateAction< WelcomeFlowExperiment > >;
```

Comment:
Similar nitpick as before, generally I think we import these types above just to make the types a bit more readable